### PR TITLE
Add cloudVPS documentation

### PR DESCRIPTION
**Pull request by fergie-nz**
Add cloudVPS documentation

### CODE REVIEW

File: documentation/Demo-Server-Guide.md [hunk 1]
```diff
@@ -269,11 +268,27 @@ From there the following commands might be useful:
 
 1. Log in to the server using terminal
 2. Copy the log to an accessible place (in the server) i.e. if getting log for instance 5000:
-`sudo cp /var/lib/docker/volumes/conforma-on-8000_logs/_data/server.log demo_server/2022-11-15.log`
-3. Now exit the server `exit` and on your local machine open the terminal 
+   `sudo cp /var/lib/docker/volumes/conforma-on-8000_logs/_data/server.log demo_server/2022-11-15.log`
```
Comment (CarlosNZ) [inline documentation/Demo-Server-Guide.md:271]: Should be a line break here (two spaces at the end, and match the indentation -- check it in Preview)

File: documentation/Demo-Server-Guide.md [hunk 2]
```diff
@@ -269,11 +268,27 @@ From there the following commands might be useful:
...
+3. Now exit the server `exit` and on your local machine open the terminal
 4. After navigate to a folder where you would like to save the log, run (some similat to this):
```
Comment (CarlosNZ) [inline documentation/Demo-Server-Guide.md:273]: Don't need "After", and typo on "similat"

File: documentation/Demo-Server-Guide.md [hunk 3]
```diff
@@ -269,11 +268,27 @@ From there the following commands might be useful:
...
-\```
-scp -i $KEY_LOC ubuntu@conforma-demo.msupply.org:/home/ubuntu/demo_server/2022-11-15.log .
-\```
+   \```
```
Comment (CarlosNZ) [inline documentation/Demo-Server-Guide.md:274]: Not sure why these quotes are escaped -- it breaks the code formatting.

File: documentation/Demo-Server-Guide.md [hunk 4]
```diff
@@ -269,11 +268,27 @@ From there the following commands might be useful:
...
 - KEY_LOC is an environment variable for the server public key :)
-- And the `.` means it will create a file with the same file name on the current folder
\ No newline at end of file
+- And the `.` means it will create a file with the same file name on the current folder
+
+## Managing CloudVPS hosted servers
+
+Conforma live servers are usually hosted on ([CloudVPS](https://www.cloudvps.com/)), an ([openstack](https://www.openstack.org/)) server provider.
+
+### Chaning CloudVPS server settings
```
Comment (CarlosNZ) [inline documentation/Demo-Server-Guide.md:285]: Typo "Chaning"

File: documentation/Demo-Server-Guide.md [hunk 5]
```diff
@@ -269,11 +268,27 @@ From there the following commands might be useful:
...
+
+Prior to changing any settings on CloudVPS, it is a good idea to take a backup snapshot of the live server, and ensure back ups are logging correctly to dropbox or wherever you are keeping your backups.
```
Comment (CarlosNZ) [inline documentation/Demo-Server-Guide.md:287]: > ~~it is a good idea~~

It is _essential_ ;)

File: documentation/Demo-Server-Guide.md [hunk 6]
```diff
@@ -269,11 +268,27 @@ From there the following commands might be useful:
...
+
+1. Log in to ([CloudVPS](https://openstack.cloudvps.com/auth/login/?next=/)). Credentials are available through Bitwarden.
+2. Access all instances through compute / instances
```
Comment (CarlosNZ) [inline documentation/Demo-Server-Guide.md:290]: Typo "compute" ?

**Review by CarlosNZ — CHANGES_REQUESTED** [review]
Just a few typos/formatting issues. Thanks

**Comment by CarlosNZ** [comment]
Also, there is an existing typo on the word "Trobleshoting" (line 7) -- can you please fix?

File: documentation/Demo-Server-Guide.md (hunk 6 above)
Comment (fergie-nz) [reply documentation/Demo-Server-Guide.md:290]: This isn't a typo, but maybe just weird language from the CloudVPS client. 

See:
https://www.cloudvps.com/knowledgebase/entry/2679-openstack-getting-started/

Comment (CarlosNZ) [reply documentation/Demo-Server-Guide.md:292]: Okay cool

**Review by CarlosNZ — APPROVED** [review]
Cool, thank you. 👍

**Pull request merged**